provider/datagen/src/transform/cldr/currency/mod.rs

```diff
@@ -129,6 +130,10 @@ fn extract_currency_essentials<'data>(
     };
 
     let mut currency_patterns_map = BTreeMap::<UnvalidatedTinyAsciiStr<3>, CurrencyPatterns>::new();
+    let mut currency_patterns_standard_none =
+        BTreeMap::<UnvalidatedTinyAsciiStr<3>, CurrencyPatterns>::new();
+    let mut currency_patterns_standard_next_to_num =
+        BTreeMap::<UnvalidatedTinyAsciiStr<3>, CurrencyPatterns>::new();
     let mut place_holders = Vec::<&str>::new();
     // A map to check if the place holder is already in the place_holders vector.
     let mut place_holders_checker_map = HashMap::<&str, u16>::new();
```

Comment:
Nit, a few lines down: the variable named `currency_pattern` is not a currency pattern; it is a set of currency placeholder values. Please pick a better variable name.

Reply:
but the`currency_pattern` is `&CurrencyPatterns` type. I do not see why do I have to change the name?